taskcluster/kinds/docker-image/kind.yml

```diff
@@ -0,0 +1,14 @@
+loader: taskgraph.loader.transform:loader
+
+transforms:
+  - taskgraph.transforms.docker_image:transforms
+  - taskgraph.transforms.cached_tasks:transforms
+  - taskgraph.transforms.task:transforms
+
+tasks:
+  browser-test:
+    symbol: I(browser-test)
```

Comment:
is this treeherder config? do we need that?

Reply:
Again, yeah this may tie into treeherder (with the symbol?) but this is what's required on Taskgraph's side to get the images built. https://taskcluster-taskgraph.readthedocs.io/en/latest/howto/docker.html#adding-image-builder-tasks